input-output-hk/mantis
# [ETCM-135] Tx history improvements for wallet

# Description

Move transaction history method to `mantis_` namespace, and add couple of additional informations a requested by Mantis Wallet devs.

It's a high priority because it's quite important from wallet perspective.

# Testing

1. all unit tests should pass
2. check if method complains about too large number of blocks within range
3. check whether method finds transactions sent or received by given address (only direct sender or direct receiver is supported so far), example tx on mordor: https://blockscout.com/etc/mordor/tx/0x6249ce7bae7e6c57b08f4a9333bd335a9c001b648e2b62b30c4cea8e80fa9842/internal-transactions, it's a tx from faucet to my testing address
